te3/tutorials/data-security/data-security-about.md

```diff
@@ -0,0 +1,386 @@
+
+_Both RLS & OLS can be easily configured, modified and tested from within Tabular Editor._
+
+---
+
+- __About Data Security and RLS/OLS (This Article):__ A functional overview of <span style="color:#01a99d">RLS</span> & <span style="color:#8d7bae">OLS</span>.
+- [__Modify/Setup an RLS Configuration:__](data-security-setup-rls.md) How to configure <span style="color:#01a99d">RLS</span> in a dataset.
+- [__Modify/Setup an OLS Configuration:__](data-security-setup-ols.md) How to configure <span style="color:#8d7bae">OLS</span> in a dataset.
+- [__Testing RLS/OLS with Impersonation:__](data-security-testing.md) How to easily validate Data Security with Tabular Editor.
+
+<br>
+
+<div class="NOTE">
+  <h5>WHY CONFIGURE ROW- OR OBJECT-LEVEL SECURITY?</h5>
+  Configuring RLS or OLS can be benificial for your model & reporting:  
+  <li> Reduce risk and improve governance by ensuring users only see data they have access to.
+  <li> Configure dynamic RLS with central role tables for consistency and lightweight maintenance.
+  <li> Have granular control over what data and objects can be queried.
+</div>
+
+---
+
+### How Does it Work?
+Data Security works at the level of the model. It is configured following the below steps:
+
+
+#### 1. __Create Roles:__ 
+_Roles_ are groups of users who have the same permission / data security logic. _Users_ in this case are identified by their email, or the email of an [Azure AD Security Group](https://learn.microsoft.com/en-us/power-bi/guidance/powerbi-implementation-planning-security-tenant-level-planning#integration-with-azure-ad). Examples of roles:
+    - Users in the same region, team or department (_EMEA_, _UA Sales Team_).
+    - Users with the same role, function or access clearance (_Key Account Managers_, _SC Clearance_).
+    - Groups defined by other business logic or arbitrary rules (_Externals_, _Build Users_). 
+
+<figure style="padding-top: 15px;">
+  <img class="noscale" src="~/images/data-security/data-security-create-role.png" alt="Data Security Create Role" style="width: 550px;"/>
+  <figcaption style="font-size: 12px; padding-top: 10px; padding-bottom: 15px; padding-left: 75px; padding-right: 75px; color:#00766e"><strong>Figure 1:</strong> In Tabular Editor, Roles are one of the top-level object types (like Tables, Relationships, etc).</figcaption>
+</figure>
+
+> [!IMPORTANT]
+> After creating a new Role in Tabular Editor, you must first set the `Model Permission` property to `Read`. 
+
+#### 2. __Specify Rules:__ 
+_Rules_ are applied for each role to one or more objects, depending on the security type:
+- _<span style="color:#01a99d">RLS Table Permissions:</span>_ DAX table expressions -- return each row evaluating `True`. These permissions traverse relationships; the design of the model is imperative to good RLS rules.
```

Comment:
Would suggest addding BOLD to: the design of the model is imperative to good RLS rules

Reply:
Agree!